marketplacetests/tests/test_marketplace_create_confirm_pin.py

```diff
@@ -13,22 +13,18 @@ class TestMarketplaceCreateConfirmPin(MarketplaceGaiaTestCase):
 
     def test_create_confirm_pin(self):
 
-        app_name = 'Test Zippy With Me'
         pin = '1234'
-        acct = FxATestAccount(base_url=self.base_url).create_account()
-
-        if self.apps.is_app_installed(app_name):
-            raise Exception('The app %s is already installed.' % app_name)
 
         marketplace = Marketplace(self.marionette, self.MARKETPLACE_DEV_NAME)
         home_page = marketplace.launch()
 
-        home_page.login(acct.email, acct.password)
+        app_name = home_page.find_first_paid_app()
```

Comment:
Calling this causing us to perform the search twice, the second search is by app name (are we certain this will be unique?). Could we just perform the `:paid` search within the test?